packages/integration-tests/suites/replay/errorResponse/test.ts

```diff
@@ -22,12 +22,14 @@ sentryTest('errorResponse', async ({ getLocalTestPath, page }) => {
   const url = await getLocalTestPath({ testDir: __dirname });
   await page.goto(url);
 
+  await waitForReplayRequest(page);
   await page.click('button');
-  await page.waitForTimeout(300);
 
   expect(called).toBe(1);
 
   // Should immediately skip retrying and just cancel, no backoff
+  // This waitForTimeout call should be okay, as we're not checking for any
+  // further network requests afterwards.
```

Comment:
The below is `DEFAULT_FLUSH_MAX_DELAY + 1`?

Reply:
I think the idea here is just to wait for a flush cycle to ensure that nothing is sent anymore after the 400 response. Can't really use the constant here as it is not exported from the replay package. But I can add another constant here